x/pairing/types/params.go

```diff
@@ -12,7 +12,7 @@ var _ paramtypes.ParamSet = (*Params)(nil)
 
 var (
 	KeyMintCoinsPerCU             = []byte("MintCoinsPerCU")
-	DefaultMintCoinsPerCU sdk.Dec = sdk.NewDecWithPrec(1, 1) // 0.1
+	DefaultMintCoinsPerCU sdk.Dec = sdk.ZeroDec()
```

Comment:
we can delete this from the protobuff alltogether

Reply:
done. see https://github.com/lavanet/lava/commit/eca2852a8a3ff3cf7e4f1c9c3351591ed83ddc03